rero_ils/modules/items/api/circulation.py

```diff
@@ -1191,7 +1192,12 @@ def _list_obj():
                 LoanState.ITEM_AT_DESK,
                 LoanState.ITEM_IN_TRANSIT_FOR_PICKUP
             ]).source(['pid'])
-        return query.count() if count else _list_obj()
+        if pids:
+            return [hit.pid for hit in query.execute()]
```

Comment:
you can use `query.scan()` instead of `execute()`. It is more efficient.

Reply:
Can you also remove the two `print` line 1180 even if it is not a part of you modifications? Thanks.

Reply:
done